data/models/changes/acteur_rgpd_anonymize.py

```diff
@@ -0,0 +1,62 @@
+"""Special change model dedicated to RGPD because:
+
+- NORMALLY we version data through RevisionActeur
+   + consequence: we create a Revision if it doesn't exist
+
+- HOWEVER WITH RGPD we don't do data versioning, we overwrite
+   the data so it disappears from our DB
+    = consequence: we don't create a Revision if it doesn't exist
+      (again we are not versioning, just overwriting)
+
+Since the approach to RGPD should be consistent, we don't
+expect the model to take any other input than the ID of the acteur
+we are changing, and the model takes care of the rest
+"""
+
+from datetime import datetime, timezone
+
+from data.models.changes.acteur_abstract import ChangeActeurAbstract
+from qfdmo.models import Acteur, ActeurStatus, RevisionActeur
+
+VALUE_ANONYMIZED = "ANONYMISE POUR RAISON RGPD"
+ACTEUR_FIELDS_TO_ANONYMIZE = {
+    "nom": VALUE_ANONYMIZED,
+    "nom_officiel": VALUE_ANONYMIZED,
+    "nom_commercial": VALUE_ANONYMIZED,
+    "email": None,  # due to email constraint
```

Comment:
Pour être compatible avec la PR sur l'interdiction de valeur Null en DB
```suggestion
    "email": "",  # due to email constraint
```